src/components/entries/Collapsible.js

```diff
@@ -64,7 +65,17 @@ export default function CollapsibleEntry(props) {
         open ? 'open' : ''
       ) }>
         {
-          entries.map(e => e.component)
+          entries.map(entry => {
+            const {
+              component: Component,
+              id
+            } = entry;
+
+            return <Component
+              { ...entry }
+              key={ id }
+              element={ element } />;
+          })
```

Comment:
With this one, we could actually get rid of setting the id twice (on the entry, on the component), by defaulting the component id with the entry id.

Example: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/bpmn/properties/IdProps.js#L65

Reply:
It's not needed, but just something that came to my mind.

Reply:
True, actually. Let me have a look and see how much effort that would be.

Reply:
@pinussilvestrus I've looked into it and we actually have to set different IDs for the entry and the actual DOM element in the future (e.g. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/spike-linting/src/provider/zeebe/properties/TaskDefinitionProps.js#L133). The IDs of entries have to be stable (e.g. `task-definition-type`) while the IDs of the DOM elements have to be dynamic (e.g. `extensionElements.0.type`) for them to be addressable. So let's keep it as it is.